nixos/tests/k3s/default.nix

```diff
@@ -17,4 +17,7 @@ in
   ) allK3s;
   single-node = lib.mapAttrs (_: k3s: import ./single-node.nix { inherit system pkgs k3s; }) allK3s;
   multi-node = lib.mapAttrs (_: k3s: import ./multi-node.nix { inherit system pkgs k3s; }) allK3s;
+  kubelet-config = lib.mapAttrs (
```

Comment:
I'd prefer keeping alphabetical order. And yes, `single-node` is at wrong order.